CallAutomation_OutboundCalling/src/app.ts

```diff
@@ -0,0 +1,219 @@
+import { config } from 'dotenv';
+import fs from "fs";
+import express, { Application } from 'express';
+import { PhoneNumberIdentifier } from "@azure/communication-common";
+import { CallAutomationClient, CallConnection, CallMediaRecognizeDtmfOptions, CallLocator, StartRecordingOptions, FileSource, CallInvite } from "@azure/communication-call-automation";
+import path from 'path';
+
+config();
+
+const PORT = process.env.PORT;
+const app: Application = express();
+app.use(express.static('webpage'));
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+
+let callConnectionId: string;
+let recordingId: string;
+let callConnection: CallConnection;
+let serverCallId: string;
+let callee: PhoneNumberIdentifier;
+let acsClient: CallAutomationClient;
+let recordingStream: NodeJS.ReadableStream;
+
+async function createAcsClient() {
+	const connectionString = process.env.CONNECTION_STRING || "";
+	acsClient = new CallAutomationClient(connectionString);
+	console.log("Initialized ACS Client.");
+}
+
+async function createOutboundCall() {
+	const callInvite: CallInvite = {
+		targetParticipant: callee,
+		sourceCallIdNumber: {
+			phoneNumber: process.env.ACS_RESOURCE_PHONE_NUMBER || "",
+		},
+	};
+
+	console.log("Placing outbound call...");
+	acsClient.createCall(callInvite, process.env.CALLBACK_URI + "ongoingcall");
+}
+
+async function startRecording() {
+	try {
+		const callLocator: CallLocator = {
+			id: serverCallId,
+			kind: "serverCallLocator",
+		};
+
+		const recordingOptions: StartRecordingOptions = {
+			callLocator: callLocator,
+		};
+
+		const response = await acsClient.getCallRecording().start(recordingOptions);
+
+		recordingId = response.recordingId;
+	} catch (error) {
+		console.error("Error starting recording:", error);
+		throw error;
+	}
+}
+
+async function playAudio(prompt: string) {
+	try {
+		const audioPrompt: FileSource[] = [{
+			url: process.env.MEDIA_CALLBACK_URI + prompt,
+			kind: "fileSource",
+		}];
+
+		await callConnection.getCallMedia().playToAll(audioPrompt);
```

Comment:
Argument of type 'FileSource[]' is not assignable to parameter of type 'FileSource'.
  Type 'FileSource[]' is missing the following properties from type 'FileSource': url, kind